src/sentry/models/auditlogentry.py

```diff
@@ -361,7 +361,11 @@ def get_note(self):
         elif self.event == AuditLogEntryEvent.SET_ONDEMAND:
             if self.data["ondemand"] == -1:
                 return "changed on-demand spend to unlimited"
-            return "changed on-demand max spend to $%d" % (self.data["ondemand"] / 100,)
+            next_ondemand_max_spend = self.data["ondemand"] / 100
+            if "prev_ondemand" in self.data:
+                prev_ondemand_max_spend = self.data["prev_ondemand"] / 100
+                return f"changed on-demand max spend to ${next_ondemand_max_spend} from ${prev_ondemand_max_spend}"
```

Comment:
Is there a function for displaying prices on the backend? If not, one could be added or something like `${next_ondemand_max_spend:.2f}` could be used for two decimal places. On the frontend, we generally show two decimal places only if there are cents in the price.

Reply:
> Is there a function for displaying prices on the backend?

I don't think there is one in sentry, as sentry doesn't have much in the way of pricing behavior. Formatting with `f"{val:.2f}` should be ok for an audit log though.

Reply:
Yeah, it'd be great to extend the audit log to getsentry eventually. This is okay for now though.